packages/theming/default-theme/src/defaultColors.win32.ts

```diff
@@ -0,0 +1,510 @@
+import { Palette, FabricWebPalette, ThemeColorDefinition } from '@fluentui-react-native/theme-types';
```

Comment:
Could you explain the difference in actual implementation between `defaultColors` and `defaultColors.macos/.win32` ?

Reply:
Yes - I should have done that in the PR description. Will update that

Reply:
Updated. Let me know if it makes sense now